mongodbatlas/data_source_mongodbatlas_team.go

```diff
@@ -18,12 +19,14 @@ func dataSourceMongoDBAtlasTeam() *schema.Resource {
 				Required: true,
 			},
 			"team_id": {
-				Type:     schema.TypeString,
-				Required: true,
+				Type:          schema.TypeString,
+				Optional:      true,
+				ConflictsWith: []string{"name"},
 			},
 			"name": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:          schema.TypeString,
+				Optional:      true,
+				ConflictsWith: []string{"team_id"},
```

Comment:
Maybe we can put them as a computed as well to set them and have/keep both in the TF state when one is chosen.